package.json

```
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@bdelab/roam-apps": "1.0.11",
-    "@bdelab/roar-firekit": "^9.2.0",
+    "@bdelab/roar-firekit": "^9.6.0-alpha.0",
```

Let's bump this to the actual `firekit` version under which we're publishing the changes

```suggestion
    "@bdelab/roar-firekit": "^9.3.0",
```